app/javascript/components/ui/Placeholder.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+
+import { classNames } from "$app/utils/classNames";
+
+type PlaceholderProps = React.PropsWithChildren<{
+  className?: string;
+  role?: string;
+  "aria-label"?: string;
+  style?: React.CSSProperties;
+}>;
+
+export const Placeholder: React.FC<PlaceholderProps> = ({ className, children, ...rest }) => {
```

Comment:
Either you export it as a single resource or your export it as default., but don't do both. Best practice is the latter, please update your imports accordingly.

```suggestion
const Placeholder: React.FC<PlaceholderProps> = ({ className, children, ...rest }) => {
```